Samples/Xamarin/HelloWorld/HelloWorld/HelloWorld.Droid/Services/MessageService.cs

```diff
@@ -0,0 +1,14 @@
+using HelloWorld.Droid.Services;
+using HelloWorld.Interfaces;
+
+[assembly: Xamarin.Forms.Dependency(typeof(MessageService))]
+namespace HelloWorld.Droid.Services
+{
+    public class MessageService : IMessageService
+    {
+        public string Message
+        {
+            get { return "Hello from Android"; }
```

Comment:
One of the three platform specific classes to test interop with DependencyService. On the first screen of the app, you should see the platform specific message above the button. Tested with Unity and Ninject.